browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1302,27 +1306,37 @@ L.CanvasTileLayer = L.Layer.extend({
 				var col = parseInt(obj.col);
 				var rowSpan = obj.rowSpan !== undefined ? parseInt(obj.rowSpan) : 1;
 				var colSpan = obj.colSpan !== undefined ? parseInt(obj.colSpan) : 1;
-				this._map._textInput.onAccessibilityFocusedCellChanged(
-					outCount, inList, row, col, rowSpan, colSpan, obj.paragraph);
+				if (typeof this._map._textInput.onAccessibilityFocusedCellChanged === 'function') {
+					this._map._textInput.onAccessibilityFocusedCellChanged(
+						outCount, inList, row, col, rowSpan, colSpan, obj.paragraph);
+				}
 			}
-			else if (textMsg.startsWith('a11yeditinginselectionstate:')) {
+			else if (textMsg.startsWith('a11yeditingix1nselectionstate:')) {
```

Comment:
typo here, added characters in the middle of a string

Reply:
This piece still remains in the patch the "a11yeditingix1nselectionstate" which doesn't look right?

Reply:
omg, unintentional char insertion, I didn't read it